.changeset/harmonious-morus-alba.md

```diff
@@ -0,0 +1,18 @@
+---
+"@comet/admin": minor
+"@comet/admin-rte": minor
+---
+
+Add a dialog option to the translation feature
+
+If enabled a dialog will open when pressing the translation button showing the original text and an editable translation
+
+ADMIN: Control if the dialog should be shown for the current scope via the `showDialog` prop
```

Comment:
```suggestion
Control if the dialog should be shown for the current scope via the `showDialog` prop
```

This is not necessary since the changeset applies to Admin packages anyway.

Reply:
3cd7ad8f21824994cb7d798428d611fa8686bf74